test/evppbe.txt

```diff
@@ -48,10 +49,11 @@ N = 1048576
 r = 8
 p = 1
 Key = 2101cb9b6a511aaeaddbbe09cf70f881ec568d574a2ffd4dabe5ee9820adaa478e56fd8f4ba5d09ffa1c6d927c40f4c337304049e8a952fbcbf45c6fa77a41a4
-#maxmem = 10000000000
 Result = SCRYPT_ERROR
+#maxmem = 10000000000
```

Comment:
What does move of #maxmem mean? That parser wouldn't tolerate it in old position?